### PR TITLE
fix: codegenning formatter in table jsx

### PR DESCRIPTION
**Pull request by kpranoto-aws**
*Description of changes:*
* using formatter in codegenned table jsx

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/codegen-ui-react/lib/react-table-renderer-helper.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+import { factory, ObjectLiteralElementLike } from 'typescript';
+
+export const objectToLiteralElements = (object: { [key: string]: string | any }): ObjectLiteralElementLike[] => {
```
Comment (letsbelopez) [inline packages/codegen-ui-react/lib/react-table-renderer-helper.ts:18]: It'd be nice to have a comment showing what the generated output code would look like for these helper functions.

**Review by letsbelopez — DISMISSED** [review]

File: packages/codegen-ui-react/lib/react-table-renderer-helper.ts (hunk 1 above)
Comment (kpranoto-aws) [reply packages/codegen-ui-react/lib/react-table-renderer-helper.ts:18]: Adding comment

Comment (letsbelopez) [reply packages/codegen-ui-react/lib/react-table-renderer-helper.ts:21]: I think it'd be easier to understand if I knew the output was for something like this
```
stringFormat: {
                  dateTimeFormat: {
                    dateFormat: \\"locale\\",
                    timeFormat: \\"hours24\\",
                  },
                },
or 
stringFormat: 'hello'
```

**Review by Jshhhh — DISMISSED** [review]

File: packages/codegen-ui-react/lib/react-table-renderer-helper.ts (hunk 1 above)
Comment (kpranoto-aws) [reply packages/codegen-ui-react/lib/react-table-renderer-helper.ts:21]: Updating

File: packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-views.test.ts.snap [hunk 2]
```diff
@@ -14,30 +14,36 @@ exports[`amplify view renderer tests should generate a table element 1`] = `
...
+                    },
+                  },
+                },
+              })}
         </TableCell>
         <TableCell>
-          {format[\\"comments\\"]
-            ? format[\\"comments\\"](item.comments)
-            : item.comments}
+          {format.comments
```
Comment (SwaySway) [inline packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-views.test.ts.snap:34]: might want to add optional chaining
```js
format?.comments ? format[item.comments(item?.comments) : formatter(item?.comments, undefined)
```

**Review by SwaySway — DISMISSED** [review]

**Review by SwaySway — APPROVED** [review]

**Review by letsbelopez — APPROVED** [review]

**Pull request merged**